Cargo.toml

```diff
@@ -25,9 +25,9 @@ frame-system-rpc-runtime-api = { path = "pallets/system/rpc/runtime-api" }
 frame-system-benchmarking = { path = "pallets/system/benchmarking" }
 
 # Polygon Primitives
-da-primitives = { git="https://github.com/maticnetwork/avail-core", version = "0.4.2", tag = "da-primitives/v0.4.2" }
-kate = { git="https://github.com/maticnetwork/avail-core", version = "0.5.3", tag = "da-primitives/v0.4.2" }
-kate-recovery = { git="https://github.com/maticnetwork/avail-core", version = "0.7.1", tag = "da-primitives/v0.4.2" }
+da-primitives = { git="https://github.com/maticnetwork/avail-core", version = "0.4.2", branch = "AV-120-rows-in-dht" }
```

Comment:
Deploy and tag `avail-core` changes before merge this.

Reply:
Sure, I have mentioned that in description, but didn't want to postpone review.

Reply:
Done, but its not possible to use different tags for different crates from same repository, I don't know whats going on but we should revisit this at some point.